### PR TITLE
Updates to lecture one. Added some info about Kipi. Some images. Fixe…

### PR DESCRIPTION
**Pull request by k1p1**
…d a broken link. Moved Game definition to the begining of the lecture.

### CODE REVIEW

**Review by Alekssasho — APPROVED** [review]

File: lectures/01.intro.md [hunk 1]
```diff
@@ -34,7 +34,7 @@
             Viktor Ketipov
         </td>
         <td>
-            <a target="_blank" href="mailto:info@kipiinteractive.com"><i class="fa fa-envelope-o"></i></a>
+            <a target="_blank" href="mailto:viktor@kipiinteractive.com"><i class="fa fa-envelope-o"></i></a>
```
Comment (nikoladimitroff) [inline lectures/01.intro.md:37]: This needs to be changed in the first slides of all lectures

File: lectures/01.intro.md [hunk 2]
```diff
@@ -128,14 +129,19 @@ We are recording, aren't we?
...
+<!-- .slide: data-background="url(resources/01.intro/cryowar_cover.png)" data-background-repeat="no-repeat" data-background-size="cover" data-background-position="center center" class="high-contrast-text" -->
 ### Kipi Interactive
 
-* Start-up working on a mobile top-down shooter
-* Founded by Viktor
-* Team of 10
-* UE4-based
+* Independent Game Development Studio
+* Founded by Viktor in 2014
+* Working on Realtime Multiplayer Arena Brawler <a target="_blank" href="https://cryowar.com">Cryowar</a>
```
Comment (nikoladimitroff) [inline lectures/01.intro.md]: 1. on **a** Realtime...
2. use the markdown syntax for links: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links

File: lectures/01.intro.md [hunk 3]
```diff
@@ -333,7 +378,7 @@ The game is constantly played, tested, changed
 
 # Why make games as business?
 
-It's ~~not~~ about the money
+Games do ~~not~~ earn a lot
```
Comment (nikoladimitroff) [inline lectures/01.intro.md:362]: I prefer the old statement but won't insist on changing it

**Review by nikoladimitroff — APPROVED** [review]
You've added 5 images but I only see 2 being used in the slides, delete the others

File: lectures/01.intro.md (hunk 3 above)
Comment (k1p1) [reply lectures/01.intro.md:362]: I think the previous version can be interpreted as "GameDev is ONLY about money".
The change tries to convey the information that there are a lot of money in Games, without specifying any agenda.

File: lectures/01.intro.md (hunk 1 above)
Comment (k1p1) [reply lectures/01.intro.md:37]: I will do a separate PR with that.

Comment (nikoladimitroff) [reply lectures/01.intro.md:37]: let me help you with that, focus on the UE4 intro slides

File: lectures/01.intro.md (hunk 3 above)
Comment (nikoladimitroff) [reply lectures/01.intro.md:362]: fair enough

**Pull request merged**